linopy/io.py

```diff
@@ -311,9 +311,13 @@ def integers_to_file(
         f.writelines(batch)
 
 
-def to_lp_file(m: Model, fn: Path, integer_label: str, slice_size: int = 10_000_000):
-    log = m._xCounter > 10_000
-
+def to_lp_file(
+    m: Model,
+    fn: Path,
+    integer_label: str,
+    slice_size: int = 10_000_000,
+    log: bool = None,
```

Comment:
```suggestion
    log: bool = True,
```
Can we make this `True` by default? It will be overwritten by the `to_file` call anyway, so `None` is not needed here?